tfhe/src/core_crypto/algorithms/lwe_multi_bit_programmable_bootstrapping.rs

```diff
@@ -1581,14 +1526,6 @@ pub fn std_multi_bit_deterministic_blind_rotate_assign<Scalar, InputCont, Output
         multi_bit_bsk.polynomial_size(),
     );
 
-    assert_eq!(
```

Comment:
why remove ?

Reply:
The input is already modulus switched (so we don't have it's initial ciphertext_modulus)
And it's ot necessary to have the same input and output moduli